.gitignore

```diff
@@ -65,21 +65,16 @@ dkms.conf
 
 # Compiled Dynamic libraries
 
-# Fortran module files
-*.mod
+# Fortran module files#
 *.smod
 
 # Compiled Static libraries
 *.lai
 
 # Executables
 
-# Google Test output files
-CMakeFiles
-cmake_install.cmake
-CMakeCache.txt
-example_test
-tests/*/Makefile
+# Directories
+build
```

Comment:
"build/" and "build" is different. This line should be "build/". You should also check the difference.